From 05bdb2bb6026c3a2e536c7143b39a763ffc1225f Mon Sep 17 00:00:00 2001
From: Dengke Du <dengke.du@windriver.com>
Date: Thu, 27 Sep 2018 09:09:40 +0800
Subject: [PATCH] ceph: disable pybind

New ceph version have bad support for corss compile,
so disable pybind temporarily.

Signed-off-by: Dengke Du <dengke.du@windriver.com>
---
 src/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 374cbc7..44845f8 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -777,7 +777,6 @@ if(NOT CEPH_BUILD_VIRTUALENV)
   set(CEPH_BUILD_VIRTUALENV /tmp)
 endif()
 
-add_subdirectory(pybind)
 add_subdirectory(ceph-disk)
 add_subdirectory(ceph-volume)
 add_subdirectory(ceph-detect-init)
-- 
2.7.4

